docs/_static/vdr/v1.yaml

```diff
@@ -262,7 +262,8 @@ components:
           $ref: '#/components/schemas/DIDDocument'
         currentHash:
           type: string
-          description: The hash of the document in hex format.
+          description: The hash of the document in hex format. No longer used, contents ignored.
```

Comment:
Why is it still a required field?

Reply:
Because is some languages the generated code will change. We could also just remove it, then already generated code will still work.

Reply:
I'd say just remove it for v6. Or maybe merge this PR as is, then remove the field from API for v6?

Regenerated code will cause integrations to break, but it's just the removal of 1 field.